### ISSUE TITLE
Annoying error message

### ISSUE THREAD

**Issue by aptiko** [openmeteo/enhydris#197]
We often get the following message. Not certain whether it's harmful or not.
```
Exception ignored in: <function WeakValueDictionary.__init__.<locals>.remove at0x7eff05738bf8>
Traceback (most recent call last):
  File "/opt/enhydris-openhi/venv/lib/python3.5/weakref.py", line 117, in remove
TypeError: 'NoneType' object is not callable
```

**Comment by aptiko** [comment]
This appears to be a [Python bug](https://bugs.python.org/issue29519). We are using 3.5.3 and this was apparently fixed in 3.5.4. It will probably go away when we upgrade to Debian 10.

**Comment by aptiko** [comment]
Closing this as invalid. If it persists after the Debian upgrade we'll reopen it.
